rotkehlchen/db/dbhandler.py

```diff
@@ -2502,10 +2502,18 @@ def query_owned_assets(self, cursor: 'DBCursor') -> List[Asset]:
                         if asset_id is not None:
                             results.add(Asset(asset_id).check_existence())
                     except UnknownAsset:
-                        self.msg_aggregator.add_warning(
-                            f'Unknown/unsupported asset {asset_id} found in the database. '
-                            f'If you believe this should be supported open an issue in github',
-                        )
+                        if table_name == 'manually_tracked_balances':
+                            self.msg_aggregator.add_warning(
+                                f'Unknown/unsupported asset {asset_id} found in the '
+                                f' manually_tracked_balances. Have you modified the assets DB? '
```

Comment:
Trying to improve error messages for the user I believe `manually_tracked_balances` is not information for them. Why not just

```suggestion
                                f' manually tracked balances. Have you modified the assets DB? '
```

Reply:
true. Will do that.